projects/schematics/src/shared/constants.ts

```diff
@@ -237,11 +239,26 @@ export const ORDER_DETAILS_SHIPPING_COMPONENT = 'OrderDetailShippingComponent';
 export const ITEM_COUNTER_COMPONENT = 'ItemCounterComponent';
 export const UPDATE_EMAIL_COMPONENT = 'UpdateEmailComponent';
 export const USER_ID_SERVICE = 'UserIdService';
+export const USER_CONNECTOR = 'UserConnector';
+export const USER_REGISTER_EFFECT = 'UserRegisterEffects';
```

Comment:
Leaving this comment instead from the previous one. In my opinion, it is fine to leave as is as it will be confusing to read, where the file name is user-register.effect but the class name is with an `s` for `effects`.